gems/pending/metadata/VmConfig/VmConfig.rb

```diff
@@ -850,6 +850,7 @@ def self.to_h(filename)
     end
     dataHash
   end
+  private_class_method :to_h
```

Comment:
This was definitely a mistake originally, `to_h` is a public method.  If we ever intended it to be private, we were wrong.  Please remove this change and do it in a followup PR or I'll do it, just say the word.